src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/model/PrisonerDetails.kt

```diff
@@ -1,6 +1,11 @@
 package uk.gov.justice.digital.hmpps.calculatereleasedatesapi.model
 
+import java.time.LocalDate
+
 data class PrisonerDetails(
```

Comment:
These 3 data classes (PrisonerDetails, SentenceAdjustments and SentenceTerm) represent responses from the prisonaApi